plugins/net_plugin/net_plugin.cpp

```diff
@@ -2756,7 +2756,7 @@ namespace eosio {
          fc::raw::unpack( ds, trx_id );
          if( trx_id ) {
             if (trx_in_progress_sz > def_max_trx_in_progress_size) {
-               report_dropping_trx(*trx_id, trx_in_progress_sz);
+               report_dropping_trx(ptr, trx_in_progress_sz);
```

Comment:
`ptr` is null here. For this case the only thing you can report is the transaction id. 
Seems you need either two log_failed_transactions or have it take a transaction id and a packed_transaction_ptr which might be null.

Reply:
Fixed.